.rubocop.yml

```diff
@@ -355,9 +355,6 @@ Performance/DeletePrefix:
 Performance/DeleteSuffix:
   Enabled: true
 
-Performance/OpenStruct:
```

Comment:
Shouldn't that cop have prevented use of OpenStruct?

I also see there is a `Style/OpenStructUse` cop now? If we're to remove it from the test suite might as well ensure it won't be re-introduced.

Reply:
All Performance cops are disabled for the test suite here. It would need to be explicitly enabled by path